Solutions/Marain.ContentManagement.Api.Specs/Marain.ContentManagement.Api.Specs.csproj

```diff
@@ -5,27 +5,33 @@
   <PropertyGroup>
     <IsPackable>false</IsPackable>
     <RootNamespace>Marain.ContentManagement.Specs</RootNamespace>
+
+    <!-- SpecFlow still fails to annotate generated content with the necessary comments, causing spurious warnings about missing XML doc comments -->
+    <NoWarn>CS1591</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\GetWorkflowStateHistory - Copy.feature.cs" />
```

Comment:
Don't these need deleting?

Reply:
Yes. I hadn't spotted that. Just pushed an update to remove them.